packages/library-base/src/commonMain/kotlin/io/realm/BaseRealm.kt

```diff
@@ -23,6 +25,13 @@ interface BaseRealm : Versioned {
      */
     val configuration: RealmConfiguration
 
+    /**
+     * Returns a copy of the schema of the realm at invocation time.
```

Comment:
It was just to make it explicit that it reflects the schema at invocation time, since `Realm.schema()` can yield different schemas over time. Maybe 
```
Returns an immutable schema of the realm.
```
A bit less convoluted, but still indicates that the instance doesn't change. ?

Reply:
Yes, that wording is bettter IMO.